src/components/cards/ContactCard.tsx

```diff
@@ -1,15 +1,16 @@
 type Props = {
+  title: string;
   name: string;
   position: string;
   email: string;
   phone: string;
 }
 
-export const ContactDiv = ({name, position, email, phone}: Props) => {
+export const ContactCard = ({title, name, position, email, phone}: Props) => {
 
   return(
     <div className="flex flex-col text-xl py-2">
-      <p className="font-bold uppercase">{name}</p>
+      <p className="font-bold "><span>{title}</span>{" "}<span className="uppercase">{name}</span></p>
```

Comment:
instead of first span, you can use it like this` {title}&nbsp;<span...{name}`

Reply:
Sure, however the &nbsp will apply for the names, which does not have the tittle and they wont be aligned, whereas with the {" "} every name will be aligned
![Screenshot 2024-01-24 212601](https://github.com/SU-UTB/web-ples-fe/assets/114882951/5bce8bdf-95d1-4f6e-bd27-ed68313c074f)

Reply:
This is with the {" "}
![image](https://github.com/SU-UTB/web-ples-fe/assets/114882951/93822ceb-0393-4435-b7ec-a8ced1f0e145)